### ISSUE TITLE
TypeError when looping over issuers

### ISSUE THREAD

**Issue by JorikMollie** [mollie/mollie-api-php#423]
## Specifications

  - API Version: 2.12.0

## Describe the issue

```
                     foreach ($method->issuers() as $issuer)
                        {
                        }

```
throws a TypeError with payment methods that do not have issuers.

**Comment by sandervanhooft** [comment]
I'll have a look.

**Comment by sandervanhooft** [comment]
PR prepared #424 .

I'll release as soon as I have received a "go".

**Comment by sandervanhooft** [comment]
Released [in v2.12.1](https://github.com/mollie/mollie-api-php/releases/tag/v2.12.1)
